impeller/entity/contents/content_context.cc

```diff
@@ -372,54 +372,54 @@ ContentContext::ContentContext(
     framebuffer_blend_softlight_pipelines_.CreateDefault(
         *context_, options_trianglestrip,
         {static_cast<Scalar>(BlendSelectValues::kSoftLight), supports_decal});
+  } else {
```

Comment:
Oh, I thought we were always doing this. So this is is only ever relevant on OpenGL ES and the desktop Metal (which I keep forgetting about) stuff right?

Reply:
And iOS simulator!